.github/workflows/lint.yml

```
@@ -31,6 +31,13 @@ jobs:
         with:
             node-version: '10.x'
 
+      -   name: Validate npm config
+          run: >-
+            (cd admin-dev/themes/default && npm ci)
+            && (cd admin-dev/themes/new-theme && npm ci)
+            && (cd themes && npm ci)
+            && (cd themes/classic/_dev && npm ci)
```

```suggestion
            (pushd admin-dev/themes/default && npm ci && popd)
            && (pushd admin-dev/themes/new-theme && npm ci && popd)
            && (pushd themes && npm ci && popd)
            && (pushd themes/classic/_dev && npm ci && popd)
```

You can't chain cd command like that. it'll not work

---

Why 3 commands than 2?

It is working because `()` starts a new sub shell.

Marking as resolved.

---

oh, funny, totally forget this one :scream: